### DISCUSSION TITLE
15.700.csv update

### DISCUSSION THREAD

**Discussion by robtop100** [john30/ebusd#1259] [Duplicates]
hi
Could somebody help me add entry to this file which can read current humidity from VR 700 controller? New Vailant apk can display this value ..so assume that ebus protocol is supporting this value..

**Comment by hgillijns** [comment]
Hi — yes, it’s available on the bus. I posted a working example for **VRC 700f (B7V00)** with the ID I found, including a quick `-def` test and the permanent CSV line here:

https://github.com/john30/ebusd/discussions/1482#discussioncomment-15356159

You can use that as a template and adapt it if your controller maps to a different Vaillant CSV / ID.
